src/main/java/bookbob/Main.java

```diff
@@ -0,0 +1,254 @@
+package bookbob;
+import bookbob.entity.Records;
+import bookbob.entity.AppointmentRecord;
+import bookbob.functions.CommandHandler;
+import bookbob.functions.FileHandler;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Main {
+    private static final Logger logger = Logger.getLogger( "Main.class" );
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("Welcome to BookBob, Dr. Bob!");
+
+        Scanner in = new Scanner(System.in);
+        Records records = new Records();
+        AppointmentRecord appointmentRecord = new AppointmentRecord();
+        FileHandler.initFile(records);
+        FileHandler.initFile(appointmentRecord);
+        CommandHandler commandHandler = new CommandHandler();
+        appointmentRecord.appointmentNotice();
+
+        while (true) {
+            String input = in.nextLine();
+            String[] inputArr = input.split(" ", 2);
```

Comment:
Maybe it would be a good idea to name the variable in full as Arr is not that commonly used to refer to Array.